src/draft-js-utils.ts

```diff
@@ -81,3 +82,17 @@ export function replaceHandle(editorState: EditorState, handle: string): EditorS
 
   return prependHandle(fromText(restOfTheText), handle)
 }
+
+export function logErrorAndReturnAlert(alert) {
```

Comment:
Let's put this function somewhere else as `draft-js-utils` doesn't really speak to what this function is about. Perhaps `error-handling.ts`